target/linux/mediatek/dts/mt7981b-xiaomi-mi-router-ax3000t-stock.dts

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+/dts-v1/;
+#include "mt7981b-xiaomi-mi-router-ax3000t.dtsi"
+
+/ {
+	model = "Xiaomi Mi Router AX3000T (stock layout)";
```

Comment:
Do we need this suffix `(stock layout)` here and for dts file name? The absence of suffixes means stock layout.

Reply:
All similar devices (e.g. AX6000, WR30U) have this suffix, so I assume that is how it's supposed to be.

Reply:
> All similar devices (e.g. AX6000, WR30U) have this suffix, so I assume that is how it's supposed to be.

1. I'm not a committer so my suggestions are optional;
2. Just because the AX6000, WR30U have has the suffix doesn't mean that it's the only right thing and we always should do so.

Reply:
> There is no any description or definition for this sentence "The absence of suffixes means stock layout". 

Ok, I try to explain. We have thousands of devices in the project and usually the `model` means exactly HW model printed on the device label:
```
        model = "Zbtlink ZBT-Z8102AX";
        model = "ZyXEL NWA50AX Pro";
        model = "Acelink EW-7886CAX";
        model = "Acer Predator W6";
        model = "ASUS TUF-AX4200";
        model = "ASUS TUF-AX6000";
        model = "GL.iNet GL-MT6000";
        model = "SmartRG SDG-8612";
        model = "SmartRG SDG-8614";
        model = "SmartRG SDG-8622";
        model = "SmartRG SDG-8632";
        model = "TP-Link TL-XDR4288";
        model = "TP-Link TL-XDR6086";
        model = "TP-Link TL-XDR6088";
```
That's OK, `model` = hw model itself. Without any useless suffixes.

Mediatek target has such a mess:
```
        model = "Linksys E8450 (UBI)";
        model = "Ubiquiti UniFi 6 LR v1 (U-Boot mod)";
        model = "Ubiquiti UniFi 6 LR v2 (U-Boot mod)";
        model = "Ubiquiti UniFi 6 LR v3 (U-Boot mod)";
        model = "UniElec U7623-02 eMMC (legacy loader, 512M RAM)";
        model = "Xiaomi Mi Router WR30U (112M UBI with NMBM-Enabled layout)";
        model = "Xiaomi Mi Router WR30U (stock layout)";
        model = "Xiaomi Mi Router WR30U (OpenWrt U-Boot layout)";
        model = "Xiaomi Redmi Router AX6000 (stock layout)";
        model = "Xiaomi Redmi Router AX6000 (OpenWrt U-Boot layout)"
```
The essence of the problem:
1. Device model property is used as a property for the any comments;
2. Comments (`112M UBI with NM BM-Enabled layout`, `OpenWrt U-Boot layout`, `UBI`) aren't related to the device hw model;
3. There is no consistent logic or meaning in these comments;
4. The comments carry absolutely no meaning if we don't change the bootloader.

So, my suggestion to all (especially p.1 to the project committers):
1. To define the rules (don't use model field for the comments). As it actually was before, for hundreds of devices.
2. Maybe introduce new Makefile variable, e.g. `DEVICE_LAYOUT_VARIANT` for the special cases like your PR;
3. Other ideas are welcome.

My suggestion to you (within the this PR, as the rules is not defined):
1. Keep device hw model untouched for the stock layout;
2. Keep dts file name clean (without `-stock`) for the stock layout.
Or is there some real reason to distort them?

> Also I belief file naming should reflect title naming, so files should also be renamed without ''-stock" suffix? Hope not because it will be confusing.

I mean the following for the stock variant:
`mt7981b-xiaomi-mi-router-ax3000t.dts`
`model = "Xiaomi Mi Router AX3000T";`
`compatible = "xiaomi,mi-router-ax3000t", "mediatek,mt7981";`
The file name matches with compatible string.

Reply:
New variable DEVICE_LAYOUT_VARIANT in variable looks great. IMHO new pull request with refactoring all MTK devices is required for this move.
My original comment for "-stock" suffix was about resulting sysupgrade files. For end user doesn't matter how dts files are named, but when user on download page looks for sysupgrade it can be really confusing what to choose. 
So you are suggesting to remove "-stock" suffix only for .dts file name, model and compatible, but leave as is "-stock" suffix for TARGET_DEVICE and file naming?